cert-manager/namespaces/cert-manager/Deployment-cert-manager.yaml

```diff
@@ -32,13 +32,12 @@ spec:
         - --v=2
         - --cluster-resource-namespace=$(POD_NAMESPACE)
         - --leader-election-namespace=kube-system
-        - --max-concurrent-challenges=20
```

Comment:
I think we want to keep this flag in place :)

Reply:
GitHub marks this as outdated, but I don't think it is (when looking at the overall diff). I think this is the only thing left to change aside from adding the requests back! :)